doc/update079.md

```diff
@@ -0,0 +1,75 @@
+## dPoW 0.7.9 update information
+
+- On your 3P node, update your Verus Coin's codebase to [`81e7a80`](https://github.com/VerusCoin/VerusCoin/tree/81e7a80271e230a132afb44967ea26f781439995), build it and then restart it.
+- On your main and 3P nodes, update your komodod to [`41265fb`](https://github.com/KomodoPlatform/komodo/tree/41265fb26d51e4f629b3dadcc92aa19e6867a76e), build it and then restart it
+
+### Update VSRC & KMD
+
+#### Using docker setup
+
+```bash
+cd notary_docker_3p
+git pull
+./update
+./start
+```
+
+#### Using other setup
+
+- Build Verus Coin
+
+```bash
+vrsc_commit='54e0cf2'
```

Comment:
should be `81e7a80` I think. 
https://github.com/VerusCoin/VerusCoin/releases/tag/v1.1.2